File: src/main/java/sawyern/cookiebot/models/entity/LootboxToken.java
package sawyern.cookiebot.models.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "LOOTBOX_TOKEN")
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = true)
public class LootboxToken extends DbItem {
    @ManyToOne
    @JoinColumn(name = "ACCOUNT_ID")
    private Account account;
}

File: src/main/kotlin/sawyern/cookiebot/models/LootboxToken.kt
package sawyern.cookiebot.models

import javax.persistence.*

@Entity
@Table(name = "TOKENS")
data class LootboxToken(
    @ManyToOne
    @JoinColumn(name = "ACCOUNT_ID")
    var account: Account,

    @ManyToOne
    @JoinColumn(name = "SEASON_ID")
    var season: Season
): DbItem()